### PR TITLE
pkgsi686Linux.mumble: fix build

### PR DESCRIPTION
**Pull request by raphaelr**
###### Description of changes

x86 mumble is a dependency of x64 [mumble_overlay](https://hydra.nixos.org/build/197512005).

ZHF: #199919

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by felixsinger — CHANGES_REQUESTED** [review]
Please add a description to the commit message.

File: pkgs/applications/networking/mumble/default.nix [hunk 1]
```diff
@@ -108,6 +108,17 @@ let
...
+
+    patches = [
+      # fixes 'static assertion failed: static_assert(sizeof(CCameraAngles) == 0x408, "");'
+      # when compiling pkgsi686Linux.mumble, which is a dependency of x64 mumble_overlay
+      # https://github.com/mumble-voip/mumble/pull/5850
+      # Remove with next version update
+      (fetchpatch {
+        url = "https://github.com/mumble-voip/mumble/commit/13c051b36b387356815cff5d685bc628b74ba136.patch";
+        hash = "sha256-S97UgvZXNGEJPM9yYbacHhIZEqqXiVf2MroZxWjZRBI=";
+      })
```
Comment (infinisil) [inline pkgs/applications/networking/mumble/default.nix:120]:
```
building '/nix/store/5508mgiigpyjvmlf3dqbf33c1aa4b0c6-13c051b36b387356815cff5d685bc628b74ba136.patch.drv'...

trying https://github.com/mumble-voip/mumble/commit/13c051b36b387356815cff5d685bc628b74ba136.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   890  100   890    0     0   3231      0 --:--:-- --:--:-- --:--:--  3236
error: hash mismatch in fixed-output derivation '/nix/store/5508mgiigpyjvmlf3dqbf33c1aa4b0c6-13c051b36b387356815cff5d685bc628b74ba136.patch.drv':
         specified: sha256-S97UgvZXNGEJPM9yYbacHhIZEqqXiVf2MroZxWjZRBI=
            got:    sha256-Rq8fb6NFd4DCNWm6OOMYIP7tBllufmQcB5CSxPU4qqg=
error: 1 dependencies of derivation '/nix/store/w2fplj51sbjqg7gw8kg88fxdwacgf18r-mumble-1.4.287.drv' failed to build
```

:thinking:

Comment (raphaelr) [reply pkgs/applications/networking/mumble/default.nix:120]: what the
it works for me, I don't have the slightest idea what's happening 😅 https://gist.github.com/raphaelr/7c230a1c326277cd5360933b4ab0edb2

Comment (raphaelr) [reply pkgs/applications/networking/mumble/default.nix]: I'm an idiot... I used nix-prefetch-url and forgot that fetchpatch does postprocessing m(

**Comment by SuperSandro2000** [comment]
@ofborg build mumble_overlay mumble

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**